src/Compilers/CSharp/Portable/CodeGen/EmitStatement.cs

```diff
@@ -1037,6 +1058,7 @@ private void EmitTryStatement(BoundTryStatement statement, bool emitCatchesOnly
         private void EmitCatchBlock(BoundCatchBlock catchBlock)
         {
             object typeCheckFailedLabel = null;
+            int currentCatchFilterLevel = _inCatchFilterLevel;
```

Comment:
>int currentCatchFilterLevel = _inCatchFilterLevel; [](http://example.com/codeflow?start=12&length=50)

Consider surrounding with ```#if DEBUG``` since the local is only used in an Assert.